primes.rb

```diff
@@ -1,6 +1,60 @@
 class Primes
-  def self.sum_to(limit = 100)
-    # TODO - add your prime number solution here...
-    "I'm working on it!"
-  end
+  
+	# I understand how to make this run with input from the user throught the terminal (limit = gets.chomp)
+	# But I couldn't succesfull get it to work with the .erb files
+
+	require 'prime'
+
+	def self.sum_to(limit = 100)
+		sum = 0
+		Prime.each(limit) do |prime|
+			sum += prime
+		end
+	end
+
+
+  #   This is the method I came up with to achieve the same thing
+  #   It's slower and for some reason doesn't work in this app,
+  #   even though it runs fine in it's own ruby file
+
+  # private
+	# def prime_range(number)
+	# 	sqrt = Math.sqrt(number).to_i
+	# 	primes = [2]
+
+	# 	(3..sqrt).step(2) do |i|
+	# 		composite = false
+	# 		primes.each do |prime|
+	# 			if i % prime == 0
+	# 				composite = true
+	# 				break
+	# 			end
+	# 		end
+
+	# 		if primes[primes.length - 1] < i && !composite
+	# 				primes << i	
+	# 		end
+	# 	end
+	# 	return primes
+	# end
```

Comment:
I wrote the algorithm in a separate file and it works fine, but when I moved it to primes.rb it threw up this error:

Internal Server Error
undefined method `join' for #<String:0x007fe2aa8649f0>
WEBrick/1.3.1 (Ruby/2.2.1/2015-02-26) at localhost:4567